ubg/src/pages/Game.js

```diff
@@ -29,24 +66,185 @@ export default function Game() {
     history.push(`/gameRoom/${roomId}`);
   }
 
+  const classes = useStyles();
+
   return (
     <div>
-      <Navbar/>
-            Game {gameId}
-      <AuthCheck>
+      <Navbar joinRoom={joinRoom} roomId={roomId} setRoomId={setRoomId}/>
```

Comment:
Did you change Navbar? I don't see the file in files changed

Reply:
I was moving the join room to the navbar but it looked wack so I deleted it. I missed deleting these though!